src/timessquare/handlers/v1/handlers.py

```diff
@@ -70,79 +90,143 @@ async def post_page(
     request_data: PostPageRequest,
     context: RequestContext = Depends(context_dependency),
 ) -> Page:
+    """Register a new page with a Jinja-templated Jupyter Notebook.
```

Comment:
This, on the other hand, assuming that it all does get lifted properly into the documentation, is a great argument for using docstrings, so I guess I can see the point of being consistent when you might write something large like that (although I'd probably just be inconsistent, for whatever it's worth).